src/api/app/views/webui/request/beta_show_tabs/_conversation.html.haml

```diff
@@ -7,18 +7,19 @@
         %i No description set
 .row
   .col-md-4.order-md-2.order-sm-1.mb-4#side-links
-    - if accepted_reviews.present? || declined_reviews.present? || open_reviews.present?
+    - if request_reviews.present?
       .d-flex.justify-content-between
         %h4
-          Reviews
-        = render AddReviewDropdownComponent.new(bs_request: bs_request, user: User.session,
-                                                can_add_reviews: can_add_reviews,
-                                                my_open_reviews: my_open_reviews)
+          Reviewers
+        %button.btn.btn-link{ data: { toggle: 'modal', target: '#add-reviewer-modal' }, title: 'Ask for a Review' }
+          %i.fas.fa-plus-circle.fa-lg.mr-1.text-secondary
+        = render partial: 'add_reviewer_dialog'
       = render AddReviewCollapsibleComponent.new
       .mb-4
-        = render partial: 'webui/request/beta_show_tabs/review_summary', collection: accepted_reviews, as: :review
-        = render partial: 'webui/request/beta_show_tabs/review_summary', collection: declined_reviews, as: :review
-        = render partial: 'webui/request/beta_show_tabs/review_summary', collection: open_reviews, as: :review
+        = render partial: 'webui/request/beta_show_tabs/review_summary', collection: request_reviews, as: :review
```

Comment:
What do you think it was the purpose of having them ordered by state? I would agree on a grouped and counted by state list (eg: 3 Accepted, 2 declined, 4 open).

Instead I believe it is more important to see the `historical` order because of two reasons:
- the state is shown anyway by the icon (green mark, yellow dot, red x), the history don't
- there is the possibility to ask for a review to the same user multiple time, history is still a thing in this matter, state don't

I'm open to other rationales here anyway